beets/autotag/match.py

```diff
@@ -241,8 +241,10 @@ def distance(items, album_info, mapping):
         dist.tracks[track] = track_distance(item, track, album_info.va)
         dist.add('tracks', dist.tracks[track].distance)
 
+    count_ign = len([x.ignorable for x in album_info.tracks if x.ignorable])
```

Comment:
@sampsyo that's what I was talking about: https://travis-ci.org/beetbox/beets/jobs/184632489#L859